app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -132,6 +132,8 @@ def collect_inventory_for_targets(ems, targets)
         # override this method and return an array of:
         #   [[target1, inventory_for_target1], [target2, inventory_for_target2]]
 
+        return [[ems, nil]] unless refresher_options.inventory_object_refresh
```

Comment:
If there wasn't a `collect_inventory_for_targets` method this would always use graph refresh even if it was disabled in settings.